spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonTracingAutoConfiguration.java

```diff
@@ -60,13 +62,24 @@ public SpanFactory spanFactory() {
         return NoOpSpanFactory.INSTANCE;
     }
 
+    @Bean
+    @ConditionalOnMissingBean(SnapshotterSpanFactory.class)
+    public SnapshotterSpanFactory snapshotterSpanFactory(SpanFactory spanFactory, TracingProperties properties) {
```

Comment:
I would expect a test case validating the existence of this default bean and a test case validating it's replaced by a user's custom `SnapshotterSpanFactory` bean.

Reply:
Hmm I will add them but that test feels very low level to me. I don't think we do that in many other places. The `withBean()` is called only 12 times through all configuration testing.

Reply:
I though we had more of those... Sure, it's low level, but with this addition we promise there's a `SnapshotterSpanFactory` bean that users can replace. Validating that behavior, although arguably it's validating Spring, still seems fair to me.

Reply:
It's fine to have it added. I have done that for every in this PR and the next ones.